src/main/scala/de/heikoseeberger/sbtheader/CommentStyle.scala

```diff
@@ -103,7 +103,7 @@ private final class LineCommentCreator(linePrefix: String) extends CommentCreato
   override def apply(text: String): String = {
     def prependWithLinePrefix(s: String) =
       s match {
-        case ""   => linePrefix
+        case ""   => if (!linePrefix.trim().isEmpty) linePrefix else ""
```

Comment:
We can't simply return `linePrefix.trim` because this would break a line prefix containing some characters with leading whitespaces. I only want to return an empty string if the `linePrefix` is blank (e.g. containing only whitespace characters) but return it unchanged if it has some content.